en/docs/administer/configuration-files.md

```diff
@@ -15,71 +15,34 @@ in github](https://github.com/wso2/carbon-kernel/tree/v4.4.7/distribution/kerne
+| [log4j.properties](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/log4j.properties) | `<PRODUCT_HOME>/repository/conf` |
+| [catalina-server.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/tomcat/catalina-server.xml) | `<PRODUCT_HOME>/repository/conf/tomcat` |
+| [tomcat-users.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/tomcat/tomcat-users.xml) | `<PRODUCT_HOME>/repository/conf/tomcat` |
+| [web.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/tomcat/web.xml) | `<PRODUCT_HOME>/repository/conf/tomcat` |
+| [context.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/tomcat/carbon/META-INF/context.xml) | `<PRODUCT_HOME>/repository/conf/tomcat/carbon/META-INF` |
+| [web.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/tomcat/carbon/WEB-INF/web.xml) | `<PRODUCT_HOME>/repository/conf/tomcat/carbon/META-INF` |
+| [Owasp.CsrfGuard.Carbon.properties](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/security/Owasp.CsrfGuard.Carbon.properties) | `<PRODUCT_HOME>/repository/conf/security` |
+| [authenticators.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/security/authenticators.xml) | `<PRODUCT_HOME>/repository/conf/security` |
+| [xss-patterns.properties](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/security/xss-patterns.properties) | `<PRODUCT_HOME>/repository/conf/security` |
+| [master-datasources.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/datasources/master-datasources.xml) | `<PRODUCT_HOME>/repository/conf/datasources` |
+| [axis2.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/axis2/axis2.xml) | `<PRODUCT_HOME>/repository/conf/axis2` |
+| [axis2_client.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/axis2/axis2_client.xml) | `<PRODUCT_HOME>/repository/conf/axis2` |
+| [tenant-axis2.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/axis2/tenant-axis2.xml) | `<PRODUCT_HOME>/repository/conf/axis2` |
+| [config-validation.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/etc/config-validation.xml) | `<PRODUCT_HOME>/repository/conf/etc` |
+| [email-admin-config.xml](https://github.com/wso2/carbon-kernel/blob/v4.4.6/distribution/kernel/carbon-home/repository/conf/email/email-admin-config.xml) | `<PRODUCT_HOME>/repository/conf/email` |
```

Comment:
Will these be redundant now that we have the deployment.toml? Are these .xml files something we should still mention in docs?

Reply:
@wijith7  Can you please look into this when you work on the toml configs?